source/release-notes/3.2.txt

```diff
@@ -670,6 +670,44 @@ addition to the numbers corresponding to the BSON types.
 :method:`db.collection.distinct()` method. For more information, see
 :method:`db.collection.explain()`.
 
+``keysExamined`` Statistic Corrected
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The :method:`explain()` method's output in ``executionStats`` or 
```

Comment:
I force pushed - it seems to have stomped over whatever was there before, so its now in a state reflecting the changes, rather than what kay commented on